tests/beta/queuing/test_beta_measure.py

```diff
@@ -129,17 +140,113 @@ class TestBetaProbs:
 
-        with qml._queuing.AnnotatedQueue() as q:
+        with AnnotatedQueue() as q:
             probs(wires)
 
-        assert len(q.queue) == 2
+        assert len(q.queue) == 1
 
-        obs = q.queue[0]
-        meas_proc = q.queue[1]
-        assert isinstance(obs, qml.Identity)
+        meas_proc = q.queue[0]
         assert isinstance(meas_proc, MeasurementProcess)
         assert meas_proc.return_type == Probability
 
-        assert len(q.get_info(obs)) == 1
-        assert q.get_info(obs) == {"owner": meas_proc}
-        assert q.get_info(meas_proc) == {"owns" : obs}
+
+class TestProperties:
+    """Test for the properties"""
+
+    def test_wires_match_observable(self):
+        """Test that the wires of the measurement process
+        match an internal observable"""
+        obs = qml.Hermitian(np.diag([1, 2, 3]), wires=['a', 'b', 'c'])
+        m = MeasurementProcess(Expectation, obs=obs)
+
+        assert np.all(m.wires == obs.wires)
+
+    def test_eigvals_match_observable(self):
+        """Test that the eigenvalues of the measurement process
+        match an internal observable"""
+        obs = qml.Hermitian(np.diag([1, 2, 3]), wires=[0, 1, 2])
+        m = MeasurementProcess(Expectation, obs=obs)
+
+        assert np.all(m.eigvals == np.array([1, 2, 3]))
+
+        # changing the observable data should be reflected
+        obs.data = [np.diag([5, 6, 7])]
+        assert np.all(m.eigvals == np.array([5, 6, 7]))
+
+    def test_error_obs_and_eigvals(self):
+        """Test that providing both eigenvalues and an observable
+        results in an error"""
+        obs = qml.Hermitian(np.diag([1, 2, 3]), wires=[0, 1, 2])
+
+        with pytest.raises(ValueError, match="Cannot set the eigenvalues"):
+            MeasurementProcess(Expectation, obs=obs, eigvals=[0, 1])
+
+    def test_error_obs_and_wires(self):
+        """Test that providing both wires and an observable
+        results in an error"""
+        obs = qml.Hermitian(np.diag([1, 2, 3]), wires=[0, 1, 2])
+
+        with pytest.raises(ValueError, match="Cannot set the wires"):
+            MeasurementProcess(Expectation, obs=obs, wires=qml.wires.Wires([0, 1]))
+
+    def test_observable_with_no_eigvals(self):
+        """An observable with no eigenvalues defined should cause
+        the eigvals property on the associated measurement process
+        to be None"""
+        obs = qml.NumberOperator(wires=0)
+        m = MeasurementProcess(Expectation, obs=obs)
+        assert m.eigvals is None
+
+
+class TestExpansion:
+    """Test for measurement expansion"""
+
+    def test_expand_pauli(self):
+        """Test the expansion of a Pauli observable"""
+        obs = qml.PauliX(0) @ qml.PauliY(1)
+        m = MeasurementProcess(Expectation, obs=obs)
+        tape = m.expand()
+
+        assert len(tape.operations) == 4
+
+        assert tape.operations[0].name == "Hadamard"
+        assert tape.operations[0].wires.tolist() == [0]
```

Comment:
Just a comment, also possible to use ``wires.labels == (0,)``

Reply:
Oh of course 🤦 I was so used to seeing `.tolist()` during the wires refactor, it was stuck in my head!